### PR TITLE
Updating HDRi Skybox automation to P1

### PR DESCRIPTION
**Pull request by NeilGitGud**
Signed-off-by: Neil Widmaier <nwidmaie@amazon.com>

## What does this PR do?

Updates the HDRi Null Renderer automation with P1 checks

## How was this PR tested?

Ran locally in Editor and passed:

[SUCCESS] Success: HDRi Skybox successfully created
[SUCCESS] Success: Entity has an HDRi Skybox component
[SUCCESS] Success: UNDO Entity creation success
[SUCCESS] Success: REDO Entity creation success
[SUCCESS] Success: Cubemap property set on HDRi Skybox component
[SUCCESS] Success: Exposure set to negative 5
[SUCCESS] Success: Exposure set to 0
[SUCCESS] Success: Entered game mode
[SUCCESS] Success: Exited game mode
[SUCCESS] Success: Entity is hidden
[SUCCESS] Success: Entity is visible
[SUCCESS] Success: Entity deleted
[SUCCESS] Success: UNDO deletion success
[SUCCESS] Success: REDO deletion success
Test result:  SUCCESS

Running all existing Null renderer tests:

AutomatedTesting\Gem\PythonTests\Atom\TestSuite_Main_Null_Render_Component_01.py ..........                      [100%]
================================================= 10 passed in 37.45s =================================================

AutomatedTesting\Gem\PythonTests\Atom\TestSuite_Main_Null_Render_Component_02.py ...........                     [100%]
================================================= 11 passed in 40.16s =================================================

AutomatedTesting\Gem\PythonTests\Atom\TestSuite_Main_Null_Render_Component_03.py ..........                      [100%]
================================================= 10 passed in 34.32s =================================================

### CODE REVIEW

File: AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py [hunk 1]
```diff
@@ -132,6 +132,12 @@
     'Kajiya': 2,
 }
 
+# Physical Sky Intensity Mode
+PHYSICAL_SKY_INTENSITY_MODE = {
+    'Ev100': 4,
+    'Nit': 3,
+}
+
```
Comment (smurly) [inline AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py:140]: I think this was already merged as part of physical sky component. your branch might be out of date.

File: AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py [hunk 2]
```diff
@@ -988,13 +996,30 @@ def occlusion_culling_plane(property: str = 'name') -> str:
...
             'name': 'Physical Sky',
+            'Intensity Mode': 'Controller|Configuration|Intensity Mode',
             'Sky Intensity': 'Controller|Configuration|Sky Intensity',
+            'Sun Intensity': 'Controller|Configuration|Sun Intensity',
+            'Turbidity': 'Controller|Configuration|Turbidity',
+            'Sun Radius Factor': 'Controller|Configuration|Sun Radius Factor',
+            'Enable Fog': 'Controller|Configuration|Fog|Enable Fog',
+            'Fog Color': 'Controller|Configuration|Fog|Fog Color',
+            'Fog Top Height': 'Controller|Configuration|Fog|Fog Top Height',
+            'Fog Bottom Height': 'Controller|Configuration|Fog|Fog Bottom Height',
```
Comment (smurly) [inline AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py:1022]: this is already merged as part of the physical sky component work. are these further updates?

File: AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_HDRiSkyboxAdded.py [hunk 3]
```diff
@@ -22,6 +22,12 @@ class Tests:
     cubemap_property_set = (
         "Cubemap property set on HDRi Skybox component",
         "Couldn't set Cubemap property on HDRi Skybox component")
+    exposure_set_to_negative_five = (
+        "Exposure set to negative 5",
+        "Exposure could not be set to negative 5")
```
Comment (smurly) [inline AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_HDRiSkyboxAdded.py:27]: spelling out negative seems odd also is this a float so -5.0

File: AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_HDRiSkyboxAdded.py [hunk 4]
```diff
@@ -136,36 +145,57 @@ def AtomEditorComponents_HDRiSkybox_AddedToEntity():
             AtomComponentProperties.hdri_skybox('Cubemap Texture'))
         Report.result(Tests.cubemap_property_set, get_cubemap_property == skybox_cubemap_material_asset.id)
 
+        properties = hdri_skybox_component.get_property_type_visibility()
+        for path, value in properties.items():
+            Report.info(f"'{path}' {value}")
```
Comment (smurly) [inline AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_HDRiSkyboxAdded.py:150]: this is for debugging to get the component property paths. you probably don't want to ship this

**Review by smurly — CHANGES_REQUESTED** [review]

File: AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_HDRiSkyboxAdded.py (hunk 4 above)
Comment (NeilGitGud) [reply AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_HDRiSkyboxAdded.py:150]: fixed

File: AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_HDRiSkyboxAdded.py (hunk 3 above)
Comment (NeilGitGud) [reply AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_HDRiSkyboxAdded.py:27]: fixed

File: AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py (hunk 2 above)
Comment (NeilGitGud) [reply AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py]: Fetch, pull and merge worked! fixed!

File: AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py (hunk 1 above)
Comment (NeilGitGud) [reply AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py]: Fetch, pull and merge worked! fixed!

**Review by smurly — APPROVED** [review]

**Pull request merged**